steps/reasons-for-appealing/reason-for-appealing/content.en.json

```diff
@@ -28,6 +28,9 @@
         "required": "You need to give at least one reason for appealing",
         "notEnough": "You have not said enough about why you want to appeal. Briefly explain why you disagree with DWP’s decision."
       }
+    },
+    "error": {
+      "invalid": "You've entered a special character. Delete it before continuing"
```

Comment:
Did @Mickyelvino approved this content?

Reply:
I am using the very same content that is in Production right now. Therefore, that was agreed with @Mickyelvino  already.